### DISCUSSION TITLE
Email validation - regExp

### DISCUSSION THREAD

**Discussion by sharonchen679** [ant-design/ant-design#44330] [Q&A]
Hi everyone!
Here is a small part of my react code (it works fine)

            <Form.Item name='email' 
                rules={[
                    { required: true , message: 'Email is required' },
                    { type: "email", message: 'Email is invalid' }
                ]}>
                <Input placeholder='Email' />
            </Form.Item>

I am looking for the EXACT regular expression that is being used here for the email validation.
I just need to use the same pattern in other parts of my code ( which are not Form.Item and therefore cannot accept "type: email" ) 
I couldn't find anything related to it in the documentation..
anyone?

**Comment by Wxh16144** [comment]
Is that what you're referring to https://github.com/yiminghe/async-validator/blob/b9a84a7313e8741703e65bbbf878cfa1bad5734f/src/rule/type.ts#L9

**Reply by sharonchen679** [reply]
Hi thank you for your reply! unfortunately that regex is not the correct one (it was very close though)
I've found an example that is not accepted by the "type: email" but does pass the regex test  ( test@豈.com )
also, i'm not sure this regex is indeed related to antd.
anyway, thanks again for trying!
